google-cloud-logging/clirr-ignored-differences.xml

```diff
@@ -95,4 +95,39 @@
     <method>com.google.api.core.ApiFuture update(com.google.logging.v2.UpdateExclusionRequest)</method>
     <differenceType>7012</differenceType>
   </difference>
+  <difference>
+    <className>com/google/cloud/logging/Logging</className>
+    <method>com.google.cloud.logging.CmekSettings getCmekSettings(java.lang.String)</method>
+    <differenceType>7012</differenceType>
+  </difference>
+  <difference>
+    <className>com/google/cloud/logging/Logging</className>
+    <method>com.google.api.core.ApiFuture getCmekSettingsAsync(java.lang.String)</method>
+    <differenceType>7012</differenceType>
+  </difference>
+  <difference>
+    <className>com/google/cloud/logging/Logging</className>
+    <method>com.google.cloud.logging.CmekSettings updateCmekSettings(com.google.cloud.logging.CmekSettings)</method>
+    <differenceType>7012</differenceType>
+  </difference>
+  <difference>
+    <className>com/google/cloud/logging/Logging</className>
+    <method>com.google.api.core.ApiFuture updateCmekSettingsAsync(com.google.cloud.logging.CmekSettings)</method>
+    <differenceType>7012</differenceType>
+  </difference>
+  <difference>
+    <className>com/google/cloud/logging/spi/v2/LoggingRpc</className>
+    <method>com.google.api.core.ApiFuture updateCmekSettingsAsync(com.google.cloud.logging.CmekSettings)</method>
+    <differenceType>7012</differenceType>
+  </difference>
+  <difference>
+    <className>com/google/cloud/logging/spi/v2/LoggingRpc</className>
+    <method> com.google.api.core.ApiFuture getCmekSettings(com.google.logging.v2.GetCmekSettingsRequest)</method>
```

Comment:
You might need to remove the space after `<method>` here. I'm not sure.

Reply:
done